definitions/standard/data_type/primitive.md

```diff
@@ -14,8 +14,10 @@
   ],
   "rules": [
     {
-      "regex": {
-        "pattern": "/^(?:-(?:[1-9](?:\\d{0,2}(?:,\\d{3})+|\\d*))|(?:0|(?:[1-9](?:\\d{0,2}(?:,\\d{3})+|\\d*))))(?:.\\d+|)$/"
+      "config": {
+        "Regex": {
+         "pattern": "/^(?:-(?:[1-9](?:\\d{0,2}(?:,\\d{3})+|\\d*))|(?:0|(?:[1-9](?:\\d{0,2}(?:,\\d{3})+|\\d*))))(?:.\\d+|)$/"
```

Comment:
We should switch to this config:         config: {pattern: "^-?\\d+(?:[.,]\\d+)?$"}